airmozilla/starred/tests/test_views.py

```diff
@@ -227,3 +227,53 @@ def test_anonymous_starred_events(self):
         ok_(event1.title in response.content)
         ok_(event2.title in response.content)
         ok_(event3.title not in response.content)
+
+    def test_event_pagination(self):
+        url = self.home_url
+        user = self._login(username='lisa')
+
+        events = []
+
+        for i in range(25):
+            event = self.create_event("Event Title %d" % (i + 1))
+            StarredEvent.objects.create(user=user, event=event)
+            events.append(event.id)
+
+        response = self.client.get(url)
+        eq_(response.status_code, 200)
+
+        for id in range(len(events)):
```

Comment:
In fact, you ought to be able to do the above code with a list of events and not a list of ids. E.g.

```python
for i in range(25):
    events.append(event)
...
for event in events:
    match = 'data-id="%d"' % event.id
    ...
```